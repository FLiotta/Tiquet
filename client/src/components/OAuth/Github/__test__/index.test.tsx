// Packages
import * as React from 'react';
import { render } from '@testing-library/react';

// Project
import GithubOAuth from '../index';


describe('<GithubOAuth />', () => {
  it('Matches snapshot', () => {
    const { container } = render(
      <GithubOAuth onSuccess={jest.fn()} />
    );

    expect(container.firstChild).toMatchSnapshot();
  });

  it('Receives text', () => {
    const testText = 'testText';

    const { container } = render(
      <GithubOAuth onSuccess={jest.fn()} text={testText} />
    );

    expect(container.firstChild).toHaveTextContent('testText');
  });
});