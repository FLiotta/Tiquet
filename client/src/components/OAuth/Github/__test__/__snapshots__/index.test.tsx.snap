// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GithubOAuth /> Matches snapshot 1`] = `
<div
  class="oauth-github"
>
  <i
    aria-hidden="true"
    class="oauth-github__icon fab fa-github"
  />
  <p
    class="oauth-github__text"
  >
    Auth with Github
  </p>
</div>
`;
